helm/chart-operator/values.yaml

```diff
@@ -0,0 +1,33 @@
+# Default values for chart-operator-chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
```

Comment:
Let's please align this file with https://github.com/giantswarm/aws-operator/blob/28a55852c391fce54ca13629aa99caca1fe12824/helm/aws-operator/values.yaml